modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/commands/openTagSelectionModal.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {addParams, navigate, openSelectionModal} from 'frontend-js-web';
+
+type Tag = {
+	qaId: string;
+	selectable: boolean;
+	value: string;
+};
```

Comment:
We should use `interface` here, instead of `type`. See from TS team lead: [Prefer Interfaces](https://ncjamieson.com/prefer-interfaces/)